vsphere/resource_vsphere_compute_cluster.go

```diff
@@ -196,6 +201,12 @@ func resourceVSphereComputeCluster() *schema.Resource {
 				Optional:    true,
 				Description: "When true, enables DRS to use data from vRealize Operations Manager to make proactive DRS recommendations.",
 			},
+			"drs_scale_descendants_shares": {
+				Type:         schema.TypeString,
+				Optional:     true,
```

Comment:
I think it would be appropriate to actually set the schema `Default` to `disabled`.

Reply:
Should we force `Default: disabled` if the API default is same?

Reply:
Set as:

```go
Default:      string(types.ResourceConfigSpecScaleSharesBehaviorDisabled),
```